src/test/java/com/suse/salt/netapi/calls/modules/MatchTest.java

```diff
@@ -0,0 +1,169 @@
+package com.suse.salt.netapi.calls.modules;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.any;
+import static com.github.tomakehurst.wiremock.client.WireMock.stubFor;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlMatching;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import java.net.HttpURLConnection;
+import java.net.URI;
+import java.util.Map;
+import java.util.Optional;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import com.github.tomakehurst.wiremock.junit.WireMockRule;
+import com.suse.salt.netapi.calls.LocalCall;
+import com.suse.salt.netapi.client.SaltClient;
+import com.suse.salt.netapi.client.impl.HttpAsyncClientImpl;
+import com.suse.salt.netapi.datatypes.AuthMethod;
+import com.suse.salt.netapi.datatypes.Token;
+import com.suse.salt.netapi.datatypes.target.MinionList;
+import com.suse.salt.netapi.results.Result;
+import com.suse.salt.netapi.utils.ClientUtils;
+import com.suse.salt.netapi.utils.TestUtils;
+
+/**
+ * Match module unit test
+ */
+public class MatchTest {
+
+    private static final int MOCK_HTTP_PORT = 8888;
+
+    private SaltClient client;
+
+    @Rule
+    public WireMockRule wireMockRule = new WireMockRule(MOCK_HTTP_PORT);
+
+    static final String JSON_CONFIG_MANAGE_MODE = ClientUtils.streamToString(
+            FileTest.class.getResourceAsStream("/modules/match/match_output.json"));
```

Comment:
Is there a reason why the name was chosen as `JSON_CONFIG_MANAGE_MODE `? I am only curious why it was not rather named `JSON_MATCH_OUTPUT` or similar.

Reply:
Hello Renner, 

Sorry in the delay to answer your message. I was on Vacation :)...
You are right. I will change the name and update the PR. Today....

Thank you